docs/en/flight_modes_fw/takeoff.md

```diff
@@ -26,19 +26,32 @@ Vehicles are [hand or catapult launched](#catapult-hand-launch) by default, but
 Takeoff mode (and [fixed wing mission takeoff](../flight_modes_fw/mission.md#mission-takeoff)) has two modalities: [catapult/hand-launch](#catapult-hand-launch) or [runway takeoff](#runway-takeoff) (hardware-dependent).
 The mode defaults to catapult/hand launch, but can be set to runway takeoff by setting [RWTO_TKOFF](#RWTO_TKOFF) to 1.
 
-To use _Takeoff mode_ you first switch to the mode, and then arm the vehicle.
+To use _Takeoff mode_ you first switch to the mode, and then arm the vehicle (or send the [MAV_CMD_NAV_TAKEOFF](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_TAKEOFF) command which does both).
```

Comment:
My assumption is that we have always supported the command, but ignored the position in it.